bears/rest/rstcheckBear.py

```diff
@@ -0,0 +1,34 @@
+from rstcheck import check
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+
+
+class rstcheckBear(LocalBear):
+    LANGUAGES = {"reStructuredText"}
+    REQUIREMENTS = {PipRequirement('rstcheck')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
```

Comment:
note that the asciinema is now out of synx, using "ignore_list" instead of the new parameter name.  Not an urgent problem, as other parts of the patch may also change.